packages/kit/src/runtime/server/page/render.js

```diff
@@ -172,21 +170,23 @@ export async function render_response({
 		init
 	].join('\n\n\t\t');
 
-	const body = options.amp
-		? rendered.html
-		: `${rendered.html}
-
-			${serialized_data
-				.map(({ url, body, json }) => {
-					let attributes = `type="application/json" data-type="svelte-data" data-url=${escape_html_attr(
-						url
-					)}`;
-					if (body) attributes += ` data-body="${hash(body)}"`;
-
-					return `<script ${attributes}>${json}</script>`;
-				})
-				.join('\n\n\t')}
-		`;
+	let body = rendered.html;
+	if (options.amp) {
+		if (options.service_worker) {
+			body += `<amp-install-serviceworker src="${options.service_worker}" layout="nodisplay"></amp-install-serviceworker>`;
+		}
+	} else {
+		body += `${serialized_data
+			.map(({ url, body, json }) => {
+				let attributes = `type="application/json" data-type="svelte-data" data-url=${escape_html_attr(
+					url
+				)}`;
+				if (body) attributes += ` data-body="${hash(body)}"`;
+
+				return `<script ${attributes}>${json}</script>`;
+			})
+			.join('\n\n\t')}`;
```

Comment:
```suggestion
		body += serialized_data
			.map(({ url, body, json }) => {
				let attributes = `type="application/json" data-type="svelte-data" data-url=${escape_html_attr(
					url
				)}`;
				if (body) attributes += ` data-body="${hash(body)}"`;

				return `<script ${attributes}>${json}</script>`;
			})
			.join('\n\n\t');
```

We should be able to remove the outer template string interpolation.